app/services/ITSAStatusService.scala

```diff
@@ -50,5 +52,29 @@ class ITSAStatusService @Inject()(itsaStatusConnector: ITSAStatusConnector,
     }
   }
 
+  def getITSAStatusDetail(itsaStatus: ItsaStatus, nino: String)
```

Comment:
this getITSAStatusDetail method would only be used for test-only purposes at the moment but could be used in production in future (it just makes an actual api call rather than being test-only). If I moved it into a test-only service, it would have to call a production connector, is this ok?